packages/slice-machine/lib/builders/SliceBuilder/layout/SideBar/components/ImagePreview.js

```diff
@@ -1,6 +1,7 @@
 import { memo, useState, useRef, Fragment } from "react";
```

Comment:
file has wrong name, it should be `jsx` or `tsx` is we migrate to type scirpt

Reply:
(following my comment about my try 👆 ), i'll try for that one :)

Reply:
worked fined, thanks for telling me to try again 👍  😄